docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.9"
+
+services:
+  postgres:
+    image: postgres:latest
+    container_name: programacion_6_postgres
+    restart: always
+    environment:
+      POSTGRES_DB: expense_tracker_group_5_postgres
+      POSTGRES_USER: user
+      POSTGRES_PASSWORD: password
+      TZ: "UTC-4"
+    ports:
+      - 5432:5432 
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+      - ./Api/DataBase/sql/init.sql:/docker-entrypoint-initdb.d/init.sql
```

Comment:
change the path after you move the init.sql file:

./init.sql:/docker-entrypoint-initdb.d/init.sql

Reply:
I'm sure I'll move the first file so that it meets the paths